docs/feature_flags.md

```diff
@@ -125,7 +125,51 @@ histogram (albeit via the text format). With this flag enabled, Prometheus will
 still ingest those conventional histograms that do not come with a
 corresponding native histogram. However, if a native histogram is present,
 Prometheus will ignore the corresponding conventional histogram, with the
-notable exception of exemplars, which are always ingested.
+notable exception of exemplars, which are always ingested. To keep the
+conventional histograms as well, enable `scrape_classic_histograms` in the
+scrape job.
+
+_Note about the format of `le` and `quantile` label values:_
+
+In certain situations, the protobuf parsing changes the number formatting of
+the `le` labels of all conventional histograms and the `quantile` labels of
+summaries. Typically, this happens if the scraped target is instrumented with
+[client_golang](https://github.com/prometheus/client_golang) provided that
+[promhttp.HandlerOpts.EnableOpenMetrics](https://pkg.go.dev/github.com/prometheus/client_golang/prometheus/promhttp#HandlerOpts)
+is set to false. In such a case any integer label values will be represented
+as a float, thus `1` becomes `1.0`, `2` becomes `2.0` and so on. This changes
+the identity of the resulting series in storage.
+
+The effect of this change is that alerts, recording rules and dashboards that
+directly reference label values as whole numbers such as `le="1"` will stop
+working.
+Alerts, recording rules and dashboards that use aggregations over the
+conventional buckets may show wrong results while they are evaluating time
+ranges spanning over the transition period. For example the quantile query
+`histogram_quantile(0.95, sum by (le) (rate(histogram_bucket)))` will be wrong
+because `rate` will see a drop to stale in some buckets, while it will see some
+completely new buckets. Time ranges not including the transition will be fine.
+
+Ways to deal with this change either globally or on a per metric basis:
+
+- Do nothing and accept that some queries that span the transition time will
+produce the wrong answer. This will include alerts that include range vector
+selectors such as for the `rate` function.
```

Comment:
```suggestion
selectors such as the `rate` function.
```

Reply:
I would leave out the whole last sentence. The impact is a bit more subtle, and everything is explained above already.